FBSnapshotTestCase/FBSnapshotTestCase.h

```diff
@@ -131,10 +131,22 @@
 /**
  When @c YES appends the name of the device model and OS to the snapshot file name.
  The default value is @c NO.
+
+ @see includeOSVersionInFilename
  */
 @property (readwrite, nonatomic, assign, getter=isDeviceAgnostic) BOOL deviceAgnostic;
 
 /**
+ Used in conjunction with @c deviceAgnostic, when @c YES the OS version will be included in the snapshot file name.
+ The default value is @c YES.
+
+ @attention @c deviceAgnostic needs to be set to @c YES for this to take effect
+
+ @see deviceAgnostic
+ */
+@property (readwrite, nonatomic, assign) BOOL includeOSVersionInFilename;
```

Comment:
Yeah, that is a nicer solution. I was planning on a follow up PR to this one that used more descriptive device names in the snapshot filenames, rather than just using model names, so maybe you can give me some feedback on that idea now, and I might be able to incorporate it into this?

Basically my rationale is that it isn't easy to determine if a snapshot is for an iPhone 6, or a 6 Plus from the filename. While the use of the device model (iPhone or iPad) and size dimensions give you some idea of what it is, having `iPhone6` or `iPhone6Plus` in the file name is more readable.

Would it be preferable to replace device model with more descriptive device names, or make descriptive device names a further option? I'd prefer to replace, but it would be a breaking change for everybody using `deviceAgnostic = true`.